crates/turbopack-dev-server/src/source/static_assets.rs

```diff
@@ -16,19 +16,27 @@ use super::{
 
 #[turbo_tasks::value(shared)]
 pub struct StaticAssetsContentSource {
-    pub prefix: String,
+    pub prefix: StringVc,
     pub dir: FileSystemPathVc,
 }
 
 #[turbo_tasks::value_impl]
 impl StaticAssetsContentSourceVc {
     #[turbo_tasks::function]
     pub fn new(prefix: String, dir: FileSystemPathVc) -> StaticAssetsContentSourceVc {
```

Comment:
Is the plan to remove this later?
The different here only being `String` vs `StringVc` make the naming confusing

Reply:
Yup.